pkgs/applications/accessibility/warpd/default.nix

```diff
@@ -0,0 +1,41 @@
+{ waylandBuild ? false, lib, fetchurl, stdenv, xorg, installShellFiles, cairo
```

Comment:
Can we build wayland and x at the same time?

Reply:
No. Only wayland or X11. This is controlled only by the PLATFORM environment variable - "wayland" or "macos". If other - X11.